openqa-label-known-issues

```diff
@@ -8,6 +8,14 @@ set -o pipefail -o errtrace
 # shellcheck source=/dev/null
 . "$(dirname "${BASH_SOURCE[0]}")"/_common
 
+if [[ $# -lt 1 ]]; then
+    echo "Usage: $0 https://openqahost/tests/<job_id>"
+    exit 1
+fi
+
+if [[ "$1" =~ ^https?://([^/]+)/tests/([0-9]+)$ ]]; then
```

Comment:
I suggest you move this into `label_issue()` and use the `$testurl` variable initialized there. 
```suggestion
if [[ $testurl =~ ^https?://([^/]+)/tests/([0-9]+)$ ]]; then
```

Maybe you can replace the not so nice `Need 'testurl'` there with the `Usage: …` message you added above.

Reply:
Without the assignation of `$host` in the top of the script, we also need to move `$host_url` and `client_args`, I dont think is a good idea.

Reply:
But let me try!